runcontainer_test.go

```diff
@@ -366,81 +366,6 @@ func TestRleRunReverseIterator16(t *testing.T) {
 	})
 }
 
-func TestRleRunSearch16(t *testing.T) {
```

Comment:
This test case should not be deleted.

Reply:
It relies on the old API. The function it tests has been deleted.

Reply:
The test is described as '"RunContainer16.search should respect the prior bounds we provide for efficiency of searching through a subset of the intervals"'. We do not provide this functionality anymore, and it was never used in the actual engine.